build-system/compile/compile.js

```diff
@@ -363,6 +363,7 @@ function compile(entryModuleFilenames, outputDir, outputFilename, options) {
       jscomp_off: ['unknownDefines'],
       define,
       hide_warnings_for: hideWarningsFor,
+      formatting: argv.pretty_print ? 'PRETTY_PRINT' : '',
```

Comment:
Add this for single pass as well:

https://github.com/ampproject/amphtml/blob/c4a663d0ba13d0488c6fe73c55dc8c971ac6ec0d/build-system/compile/single-pass.js#L107-L141